docs/src/known_issues.md

```diff
@@ -2,7 +2,7 @@
 
 Routine issues are posted
 [here](https://github.com/alan-turing-institute/MLJ.jl/issues). Below
-are some longer term issues and limitations.
+are some longer-term issues and limitations.
```

Comment:
"longer" is a bone fide  adjective, not a noun acting as an adjective.
```suggestion
are some longer term issues and limitations.
```

Reply:
I went with the form used by Oxford Dictionary and Wikipedia but both are used.